api/saucectl.schema.json

```diff
@@ -1330,6 +1331,7 @@
                 "$ref": "#/allOf/8/then/properties/playwright/properties/version",
                 "enum": [
                   "package.json",
+                  "1.50.1",
```

Comment:
can we also add 1.51.1?

Reply:
We add whatever was added to the runner at the start of the release process. Adding 1.51.1 now means starting all over again.

Reply:
This PR is just for the json schema which powers autocomplete for the IDEs that support it.